### PR TITLE
Add the rest of implemented GQLSTATUS error codes

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-220.surge.sh

Updated pages:
- [List of all GQLSTATUS server error codes](https://neo4j-docs-status-codes-220.surge.sh/status-codes/5/errors/gql-errors)

File: modules/ROOT/pages/errors/gql-errors.adoc [hunk 1]
```diff
@@ -543,9 +555,9 @@ Status description:: error: data exception - parsing JSON failure. Underlying er
 
 // Status description:: error: data exception - unexpected map entry. Invalid input. Unexpected key `{ $key }`, expected keys are `{ $listAllowedKeys }`.
 
-// === 22NB0
+=== 22NB0
 
-// Status description:: error: data exception - invalid property based access control rule involving WHERE and IS NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use `WHERE' syntax in combination with `IS NULL` instead.
+Status description:: error: data exception - invalid property based access control rule involving WHERE and IS NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use `WHERE' syntax in combination with `IS NULL` instead.
```
Comment (Lojjs) [inline modules/ROOT/pages/errors/gql-errors.adoc:560]: Not sure how you want to format this, but probably not with different quotes before and after WHERE. 

In the code it is ...evaluates to 'NULL'. Use `WHERE` syntax in combination with `IS NULL` ... so inconsistent (I think @renetapopova wanted to go through and fix these sort of inconsistencies), but not as broken as here in the docs.

File: modules/ROOT/pages/errors/gql-errors.adoc [hunk 2]
```diff
@@ -969,10 +1017,18 @@ Status description:: error: syntax error or access rule violation - cannot modif
...
+
+Status description:: info: accessing multiple graphs only supported on composite databases. Accessing multiple graphs in the same query is only supported on composite databases. Connect to a composite database with the desired constituents.
+
 === 42NA6
 
 Status description:: info: invalid alias target. Aliases are not allowed to target composite databases.
 
+=== 42NA7
+
+Status description:: info: referenced database not found. No database is corresponding to { $db }. Verify that the elementId is correct.
```
Comment (Lojjs) [inline modules/ROOT/pages/errors/gql-errors.adoc:1030]:
```suggestion
Status description:: info: referenced database not found. No database is corresponding to `{ $db }`. Verify that the elementId is correct.
```

File: modules/ROOT/pages/errors/gql-errors.adoc [hunk 3]
```diff
@@ -900,6 +936,10 @@ Status description:: error: syntax error or access rule violation - unexpected e
 
 Status description:: error: syntax error or access rule violation - unsupported normal form. Unknown Normal Form: `{ $input }`.
 
+=== 42N51
+
+Status description:: error: syntax error or access rule violation - invalid parameter. Invalid parameter `{ $param }`
```
Comment (Lojjs) [inline modules/ROOT/pages/errors/gql-errors.adoc:941]:
```suggestion
Status description:: error: syntax error or access rule violation - invalid parameter. Invalid parameter `{ $param }`.
```

File: modules/ROOT/pages/errors/gql-errors.adoc [hunk 4]
```diff
@@ -777,14 +793,26 @@ Status description:: error: syntax error or access rule violation - invalid use
...
+
+Status description:: error: syntax error or access rule violation - invalid use of a subquery in MERGE. Subqueries are not allowed in a MERGE clause.
+
 === 42I50
 
 Status description:: error: syntax error or access rule violation - token name too long. Invalid input `{ $input }..`. A `{ $tokenType }` name cannot be longer than `{ $maxTokenLength }`.
 
+=== 42I51
+
+Status description:: info: invalid call signature. The procedure or function `{ %s }` must have the signature: `{ %s }`.
```
Comment (Lojjs) [inline modules/ROOT/pages/errors/gql-errors.adoc:814]:
```suggestion
Status description:: info: invalid call signature. The procedure or function `{ $procFun }` must have the signature: `{ $sig }`.
```

File: modules/ROOT/pages/errors/gql-errors.adoc [hunk 5]
```diff
@@ -731,10 +743,14 @@ Status description:: error: syntax error or access rule violation - invalid case
 
 Status description:: error: syntax error or access rule violation - invalid `FOREACH`. Invalid use of `{ $clause }` inside 'FOREACH'.
 
-=== 42I02
+??? === 42I02
```
Comment (Lojjs) [inline modules/ROOT/pages/errors/gql-errors.adoc:746]: What is the questions marks here for?

**Review by Lojjs — CHANGES_REQUESTED** [review]
Found some smaller stuff, but over all it looks good

File: modules/ROOT/pages/errors/gql-errors.adoc (hunk 5 above)
Comment (NataliaIvakina) [inline modules/ROOT/pages/errors/gql-errors.adoc:746]:
```suggestion
=== 42I02
```

Comment (NataliaIvakina) [reply modules/ROOT/pages/errors/gql-errors.adoc:746]: forgot to remove them

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-220.surge.sh

Updated pages:
- [List of all GQLSTATUS server error codes](https://neo4j-docs-status-codes-220.surge.sh/status-codes/5/errors/gql-errors)

File: modules/ROOT/pages/errors/gql-errors.adoc (hunk 1 above)
Comment (NataliaIvakina) [inline modules/ROOT/pages/errors/gql-errors.adoc:560]:
```suggestion
Status description:: error: data exception - invalid property based access control rule involving WHERE and IS NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use `WHERE` syntax in combination with `IS NULL` instead.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-220.surge.sh

Updated pages:
- [List of all GQLSTATUS server error codes](https://neo4j-docs-status-codes-220.surge.sh/status-codes/5/errors/gql-errors)

File: modules/ROOT/pages/errors/gql-errors.adoc (hunk 1 above)
Comment (NataliaIvakina) [reply modules/ROOT/pages/errors/gql-errors.adoc]: _invalid property based access control rule involving WHERE and IS NULL_ is subcondition. 
It seems you don't use any kind of quotes or backticks in subconditions (look at them in the spreadsheet or code.)
In another PR, I already asked the question about the Cypher clauses and expressions mentioned in the messages. If I remember properly, it was decided to  postpone this discussion.

**Review by Lojjs — APPROVED** [review]
Looks good now

File: modules/ROOT/pages/errors/gql-errors.adoc (hunk 1 above)
Comment (Lojjs) [reply modules/ROOT/pages/errors/gql-errors.adoc]: Yes, we still have a trello card for making it more consistent. As long as the docs rendering is not broken, it should be good enough for now

**Pull request merged**